shared/people/wot-task/container.tsx

```diff
@@ -0,0 +1,38 @@
+import WotTask from '.'
+import * as PeopleGen from '../../actions/people-gen'
+import * as RPCTypes from '../../constants/types/rpc-gen'
+import * as Container from '../../util/container'
+
+type OwnProps = {
+  key: string
+  onClickUser: (username: string) => void
+  status: RPCTypes.WotStatusType
+  vouchee: string
+  voucher: string
+}
+
+const mapStateToProps = state => ({myUsername: state.config.username || ''})
+
+const mapDispatchToProps = dispatch => ({
+  onDismiss: (voucher: string, vouchee: string) => {
+    dispatch(PeopleGen.createDismissWotNotifications({vouchee, voucher}))
+  },
+})
+
+const mergeProps = (stateProps, dispatchProps, ownProps: OwnProps) => {
+  const otherUser =
+    stateProps.myUsername.localeCompare(ownProps.voucher) === 0 ? ownProps.vouchee : ownProps.voucher
+  return {
+    badged: true,
+    key: ownProps.key,
+    onClickUser: ownProps.onClickUser,
+    onDismiss: dispatchProps.onDismiss,
+    otherUser: otherUser,
+    userForIcon: otherUser,
+    vouchee: ownProps.vouchee,
+    voucher: ownProps.voucher,
+    wotStatus: ownProps.status,
+  }
+}
+
+export default Container.connect(mapStateToProps, mapDispatchToProps, mergeProps)(WotTask)
```

Comment:
We prefer not to add new connected components (i.e. `container.tsx`). Preference is to use hooks like `Container.useSelector` in place of `mapStateToProps` and `Container.useDispatch()` in place of `mapDispatchToProps`.

Some example files:

https://github.com/keybase/client/blob/master/shared/team-building/search-result/people-result.tsx

https://github.com/keybase/client/blob/master/shared/crypto/operations/encrypt.tsx